src/python_lib_osw_validation/__init__.py

```diff
@@ -0,0 +1,89 @@
+import os
+import json
+import jsonschema
+from typing import Dict, Any
+from .zipfile_handler import ZipFileHandler
+from .extracted_data_validator import ExtractedDataValidator
+
+SCHEMA_PATH = os.path.join(os.path.dirname(__file__), 'schema')
+
+
+class OSWValidation:
+    schema_dir = os.path.join(SCHEMA_PATH, 'opensidewalks.schema.json')
+
+    def __init__(self, zipfile_path: str, schema_file_path=None):
+        self.zipfile_path = zipfile_path
+        self.extracted_dir = None
+        if schema_file_path is None:
+            self.schema = self.load_osw_schema(OSWValidation.schema_dir)
+        else:
+            self.schema = self.load_osw_schema(schema_file_path)
+
+        self.error = None
+
+    def load_osw_schema(self, schema_path: str) -> Dict[str, Any]:
+        '''Load OSW Schema'''
+        try:
+            with open(schema_path, 'r') as file:
+                schema = json.load(file)
+            return schema
+        except Exception as e:
+            self.error = e
+            raise Exception(f'Invalid or missing schema file: {e}')
+
+
+
+    def validate(self) -> bool:
```

Comment:
redefine this method to return two variables, (isValid, errorList) . The list will have the errors and `isValid` will have the same bool as sent now.